### ISSUE TITLE
Concerns in bgnpcgn-bal-Arab-Latn-2008

### ISSUE THREAD

**Issue by AhMohsen46** [interscript/interscript#620]
ي is sometimes transliterated as e, othertimes as y
example as e
![image](https://user-images.githubusercontent.com/34799755/98384271-3695df00-2056-11eb-98f2-99dff63424c2.png)
example as y
![image](https://user-images.githubusercontent.com/34799755/98384492-82e11f00-2056-11eb-9dc8-a45148c39984.png)

و is sometimes transliterated as o, othertimes as w

![image](https://user-images.githubusercontent.com/34799755/98349135-4c89ac80-2022-11eb-9afc-d6f010fad182.png)
![image](https://user-images.githubusercontent.com/34799755/98349184-5c08f580-2022-11eb-914e-ec1c4cd88dc4.png)

**Comment by ronaldtse** [comment]
Clarification sought. Thanks @AhMohsen46 !

**Comment by ronaldtse** [comment]
Received clarification from BGN:

> These do fall under the same field of questions as the previous systems.
>
> It is difficult to set down rules for those ambiguous sets. As in our English language, we have the letter "y", in certain spellings, is treated differently. ​Examples: 'You' versus 'sYmbol'. 
>
> Then we have words that have "silent" letters (this brings to mind the "و" in خوا, that is transliterated "v" and not "u+macron" in "khva" - 'a+macron', but pronounced "kha" - 'a+macron'). English examples: pneumonia, aisle, tsunami... 
> 
> Each language system, though some may appear similar, would need their own specific set of rules. As they may have discovered, rules in Persian do not/cannot be applied with the same expectations to Dari, Pashto. For the Perso-Arabic scripts, the letters with multiple values (ا, و, ی, ) will need special consideration depending on where they occur (independent, initial, medial, or final), as well as when combined with other letter/characters.”

@AhMohsen46 I believe this means that the transliteration produced by this system is meant to -disambiguate the underlying pronunciation (just like English), so we will need to have the proper rules to differentiate them. Could you help find those specific rules and conditions?

**Comment by AhMohsen46** [comment]
@ronaldtse yes, this makes sense, I will try to check if there is something specific for this, or if we should search for phonetics of the words, not sure if there is something like that, but will take a look, and tell you what I find
